src/frontend/komponenter/Sak/SakListe.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { useSakListeRessurser } from '../../context/SakListeContext';
+import { RessursStatus } from '../../typer/ressurs';
+import styled from 'styled-components';
+import { useHistory } from 'react-router';
+
+const Td = styled.td`
+    border: 1px solid black;
+`;
+
+const Th = styled.th`
+    border: 1px solid black;
+`;
+
+const SakListe: React.FunctionComponent = () => {
+    const history = useHistory();
+    const { ressurser, hentSakListe } = useSakListeRessurser();
+    React.useEffect(() => {
+        // I SakContainer brukes ressurser.sak.status !== RessursStatus.SUKSESS men då får jeg en evig loop
+        if (ressurser.saker.status === RessursStatus.IKKE_HENTET) {
+            hentSakListe();
+        }
+    });
```

Comment:
```suggestion
    }, []);
```

Dette bør vel gjøre susen?